spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AppRegistryControllerTests.java

```diff
@@ -154,6 +154,24 @@ public void testListApplicationsByType() throws Exception {
 	}
 
 	@Test
+	public void testListApplicationsBySearch() throws Exception {
+		mockMvc.perform(get("/apps?search=timestamp").accept(MediaType.APPLICATION_JSON)).andDo(print()).andExpect(status().isOk())
```

Comment:
we can remove the print().  I can do that on merge.

Reply:
Indeed there is a lot of other tests to log these as well. Feel free to remove those as well as we don't need to be so noisy.